prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctionsTests.kt

```diff
@@ -646,103 +612,51 @@ class CustomFHIRFunctionsTests {
-            mutableListOf(DateType(randomDate)),
-            mutableListOf(mutableListOf(DateType(randomDate)))
-        )
-
-        val age = ageList[0]
-        assertThat(age is Age).isEqualTo(true)
-        if (age is Age) {
-            assertThat(age.unit).isEqualTo("day")
-            assertThat(age.value.toInt()).isEqualTo(0)
-            assertThat(age.code).isEqualTo("d")
-        }
-    }
-
-    @Test
-    fun `test convertDateToAge 4 days`() {
-        val currentDate = Date()
-        val calendar = Calendar.getInstance()
-        calendar.timeInMillis = currentDate.time
-        // has to be at least two for the purposes of this since we are getting the current time and time passes when we
-        // run it
-        calendar.add(Calendar.DAY_OF_YEAR, -4)
-
-        val ageList = CustomFHIRFunctions.convertDateToAge(
-            mutableListOf(DateType(calendar.time)),
-            mutableListOf(mutableListOf(StringType("day")))
-        )
-
-        val age = ageList[0]
-        assertThat(age is Age).isEqualTo(true)
-        if (age is Age) {
-            assertThat(age.unit).isEqualTo("day")
-            assertThat(age.value.toInt()).isEqualTo(4)
-            assertThat(age.code).isEqualTo("d")
-        }
-    }
-
-    @Test
-    fun `test convertDateToAge 4 days - default`() {
-        val currentDate = Date()
-        val calendar = Calendar.getInstance()
-        calendar.time = currentDate
-        // has to be at least two for the purposes of this since we are getting the current time and time passes when we
-        // run it
-        calendar.add(Calendar.DAY_OF_YEAR, -4)
-
-        val ageList = CustomFHIRFunctions.convertDateToAge(
-            mutableListOf(DateType(calendar.time)),
-            mutableListOf()
+        val someDate = Date(1687464192857)
+        val results = listOf(
+            // Explicitly look for "day"
+            CustomFHIRFunctions.convertDateToAge(
+                mutableListOf(DateType(someDate)),
+                mutableListOf(mutableListOf(DateType(someDate), StringType("day")))
+            ),
+            // Get default response without specifying type
+            CustomFHIRFunctions.convertDateToAge(
+                mutableListOf(DateType(someDate)),
+                mutableListOf(mutableListOf(DateType(someDate)))
+            )
         )
-
-        val age = ageList[0]
-        assertThat(age is Age).isEqualTo(true)
-        if (age is Age) {
-            assertThat(age.unit).isEqualTo("day")
-            assertThat(age.value.toInt()).isEqualTo(4)
-            assertThat(age.code).isEqualTo("d")
+        results.forEach { ageList ->
+            val age = ageList[0]
+            assertThat(age is Age).isEqualTo(true)
+            if (age is Age) {
+                assertThat(age.unit).isEqualTo("day")
+                assertThat(age.value.toInt()).isEqualTo(0)
+                assertThat(age.code).isEqualTo("d")
+            }
         }
     }
 
```

Comment:
We should probably update the name of the below test.

Reply:
This also applies for some of the combined tests above, don't need things like `default` in the test names anymore for the most part.